### PR TITLE
New Logo!

### PR DESCRIPTION
**Pull request by alanlgirard**
New logos everywhere!  The new logo was replaced on the Index and About pages.  Because the word "Automation" in the logo has the default corporate color of grey, the toolbar color needed to change.  The color pallet grey "900" color was replaced with black (#000000). This color change causes the toolbars and buttons to have a default color of black.  

- Replaced Index logo
- Replaced About page logo
- Replaced favicon with lightbulb without the globe
- Changed color theme default to black

Index
image

About Page 
image

- [x] Code Review 1 @McTalian
- [x] Code Review 2 @davethenipper
- [x] Functional Review 1 @McTalian
- [x] Functional Review 2 @bcbrennecke

@Mctalian @YashdalfTheGray @bcbrennecke @monotkate @LongArmMcGee @davethenipper

### CODE REVIEW

**Comment by alanlgirard** [comment]
Closes issue #279

**Comment by Mctalian** [comment]
What about the "Learn" page? Do we want to change that logo as well?

File: src/index.html [hunk 1]
```diff
@@ -78,11 +78,11 @@
                 <i class="material-icons icon-button-adjust">&#xE5D2;</i>
             </md-button>
             <span class="toolbar-lightbulb">
-                <img ui-sref="home"src="assets/Innovation_Challenge_Bulb.png" width="25"/>
+                <img ui-sref="home"src="assets/Innovate-Rockwell-Automation-White.png" width="250"/>
                 <md-tooltip md-direction="bottom">Home</md-tooltip>
             </span>
             <div ng-controller="SidenavCtrl">
-            <md-button class=" toolbar-left-link" aria-label="Add another idea" ng-click="navTo('addidea')"><md-button aria-label="Add another idea" ng-click="navTo('addidea')">
+            <md-button aria-label="Add another idea" ng-click="navTo('addidea')"><md-button aria-label="Add another idea" ng-click="navTo('addidea')">
```
Comment (Mctalian) [inline src/index.html]: There's an `md-button` within an `md-button` here. I know that's not you. Probably a mis-merge by someone else.

Can we add "toolbar-links" class to the Create md-button? I like the highlighting and ripple container expanding from top of toolbar to bottom.

**Comment by Mctalian** [comment]
Approved pending my comments.

**Comment by alanlgirard** [comment]
For the Learn page I just replaced the lightbulb logo.  If people don't like this I can use the actual logo. 

image

**Comment by Mctalian** [comment]
:+1: Thanks for the quick fixes

**Comment by monotkate** [comment]
I'm not entirely sold on solid black.....

Logo looks great, although I'd bump a bit more of a buffer between it and the Create button.  They look a tad scrunched.

I might add more spacing on the about page too.  Looks a tad busy right now, and we've got space to play.  It took me a few mins to actually find the about page, lol.  We might want to move it to the Learn page *random thought*

Also noticing (has nothing to do with this PR) where did the longest tooltips known to man come from on the about page.  @YashdalfTheGray back me up on this.

**Comment by bcbrennecke** [comment]
Woah my bad, I thought I approved this.  I'm not sold on the black either, but I let's get this in there and we can talk about that at a later time.
@monotkate Did you ever see the tooltips for the browse page when they were the description of the card?
Looks great!

**Comment by Mctalian** [comment]
@monotkate, yeah those are totally not the longest known to man, as
@bcbrennecke mentioned.

Secondly, They've been there since I put them there when the About Page was
redesigned. You don't need backup, it's not a new thing, there are just
much bigger fish to fry. It needs to go on the backlog, not a conversation
for this thread.

**Comment by YashdalfTheGray** [comment]
I am going to back @monotkate on this. Those tooltips need to be fixed, I don't care where they are. But we already have a mountain for tech debt, pile it on.

**Comment by monotkate** [comment]
I opened an issue on the tooltips.  The color of the toolbar does relate to this issue though.

**Comment by monotkate** [comment]
@bcbrennecke if you don't actually approve, uncheck your box

**Comment by bcbrennecke** [comment]
I do approve.  We've already talked that a redesign is coming.  I'm sure there will be plenty of discussion about color at that point, we don't need to hash it out now.  Would be a waste of all our time just to do work twice.
Don't hold up this PR due to the tooltips, @monotkate opened #313 for that.
Let's get this merged.

**Comment by alanlgirard** [comment]
Ok made the small changes of increasing the padding for the toolbar buttons and the padding around the about page logo.  Also changed the color palette to use the default grey color of #212121 which is less harsh on the eyes.  We can discuss other color options etc when we do the homepage layout redesign.

image

**Comment by monotkate** [comment]
Approoooove while I've got service

**Pull request merged**